### PR TITLE
Arrow up and down to change the address

### PR DESCRIPTION
**Pull request by jibin2706**
#6

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -34,6 +35,50 @@ class IPAddress extends Component {
...
+          const newOctet = [...prevState.octets];
+          const index = event.target.dataset.octet;
+          newOctet[index] = newOctet[index] + 1;
+          if (newOctet[index] > 0 && newOctet[index] < 256)
+            return { octets: newOctet };
+        });
+      }
+    }
+  }
+
```
[the remark this answers is not in the document]
Comment (jibin2706) [reply src/index.js]: Yes much better solution.

**Comment by jibin2706** [comment]
Changes made @yuvadm

**Comment by yuvadm** [comment]
@jibin2706 Looks much better! Do you want to add mouse scroll support as well or is it more complicated?

**Comment by jibin2706** [comment]
@yuvadm Seems complicated.

**Pull request merged**